fixtures/pages-simple-assets/tests/index.test.ts

```diff
@@ -1,21 +1,18 @@
 import { resolve } from "node:path";
 import { fetch } from "undici";
-import { describe, it, beforeAll, afterAll } from "vitest";
+import { describe, it, afterAll } from "vitest";
 import { runWranglerPagesDev } from "../../shared/src/run-wrangler-long-lived";
 
-describe.concurrent("Pages Functions", () => {
-	let ip, port, stop;
-
-	beforeAll(async () => {
-		({ ip, port, stop } = await runWranglerPagesDev(
-			resolve(__dirname, ".."),
-			"public",
-			[]
-		));
+describe("Pages Functions", async () => {
+	const { ip, port, stop } = await runWranglerPagesDev(
+		resolve(__dirname, ".."),
+		"public",
+		[]
+	);
```

Comment:
Here is a good reason to avoid this. The body of the describe gets run even if you are skipping this test. I just try it by changing it to `describe.skip()` and you can see the code is still run.